src/Parser/FireMage/Modules/Features/AlwaysBeCasting.js

```diff
@@ -35,7 +35,7 @@ class AlwaysBeCasting extends CoreAlwaysBeCasting {
 
     when(deadTimePercentage).isGreaterThan(0.2)
       .addSuggestion((suggest, actual, recommended) => {
-        return suggest(<span>Your downtime can be improved. Try to Always Be Casting (ABC), try to reduce the delay between casting spells. Even if you have to move, try casting instants, even unbuffed <SpellLink id={SPELLS.ICE_LANCE_CAST.id} /> spam is better than nothing.</span>)
+        return suggest(<span>Your downtime can be improved. Try to Always Be Casting (ABC), try to reduce the delay between casting spells. If you have to move, try casting <SpellLink id={SPELLS.FIRE_BLAST.id} /> or <SpellLink id={SPELLS.SCORCH.id} />.</span>)
```

Comment:
As Fire Blast isn't on the GCD, casting it while moving won't actually make any difference to your ABC stat.